packages/admin/cms-admin/src/dam/DamTable.tsx

```diff
@@ -39,9 +39,12 @@ import EditFile from "./FileForm/EditFile";
 
 interface FolderProps extends DamConfig {
     filterApi: IFilterApi<DamFilter>;
+    additionalToolbarItems?: AdditionalToolbarItem[];
```

Comment:
I'd make this prop even more flexible by using `React.ReactNode`. Doing so you don't need to iterate the prop

Reply:
I am not sure if I understand you correctly. Should I completely remove the type `AdditionalToolbarItem` and replace all occurrences of `additionalToolbarItems?: AdditionalToolbarItem[];` with `additionalToolbarItems?: React.ReactNode`?

Reply:
Yes

Reply:
But a `React.ReactNode` can also be  `boolean | null | undefined` - so I'd have to check the type anyway or am I wrong?

Reply:
Why? You could simply render the `additionalToolbarItems`, no matter what type they are. If someone decides to render a boolean, so be it 🙂

Reply:
Ok